python/README.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Looks like an accidental change made it in?

Reply:
Oops, thanks!